apps/crn-frontend/src/network/users/Research.tsx

```diff
@@ -31,7 +32,7 @@ const Research: React.FC<ResearchProps> = ({ user }) => {
         {...user}
         userProfileGroupsCard={
           <Frame title={null} fallback={null}>
-            <GroupsCard user={user} />
+            <GroupsCard user={user} groups={groups} />
```

Comment:
With the prev implementation there were two calls being made to "/groups", fixed that